### PR TITLE
Test for git

### PR DESCRIPTION
**Pull request by gitmapd**
Added test for git and replaced "which", there are distributions that "which" doesn't exist, it's not portable.

### CODE REVIEW

**Comment by gitmapd** [comment]
I forgot to change the shebang to #!/usr/bin/env bash .

File: install.sh [hunk 1]
```diff
@@ -18,6 +18,18 @@ echo "Installing Canasta CLI"
...
+GIT_IS_AVAILABLE=$?
+
+if [ $GIT_IS_AVAILABLE -ne 0 ]; 
+then echo "Git was not found, please install before continuing.";
+     exit; 
+else
+     echo "Git was found on the system"
+fi
+
 loc=$(which docker)
```
Comment (jeffw16) [inline install.sh:33]: `which` and the lines below it that use the `$loc` variable don't seem to be removed from here.

**Review by jeffw16 — DISMISSED** [review]

**Comment by gitmapd** [comment]
Hi, I apologize but i don't get your sentence. Thanks.

File: install.sh (hunk 1 above)
Comment (jeffw16) [reply install.sh:33]: You said you replaced `which` because it's not portable, but your new code still contains `which`.

**Comment by gitmapd** [comment]
I know i i think you got to see the comment before I removed it, but Yaron
told me to make a new patch for it. There is another patch to change the
shebang, do you want me to make a patch with which + the shebang changed?

**Comment by yaronkoren** [comment]
Note: this fixes issue #26.

**Review by bawolff — APPROVED** [review]
If the point is to make this posix compliant, this doesn't accomplish it (as the docker check still uses which), but as a step in the right direction seems reasonable

**Review by yaronkoren — APPROVED** [review]

File: install.sh (hunk 1 above)
Comment (yaronkoren) [reply install.sh:33]: These comments are not relevant to the actual code additions, I think - just to the commit summary.

**Pull request merged**